pkg/agent/client.go

```diff
@@ -128,6 +130,8 @@ func GenAgentIdentifiers(addrs string) (Identifiers, error) {
 			agentIDs.Host = append(agentIDs.Host, ids...)
 		case CIDR:
 			agentIDs.CIDR = append(agentIDs.CIDR, ids...)
+		case DefaultRoute:
+			agentIDs.DefaultRoute = true
```

Comment:
We shouldn't assume that ids == "true". If someone tries to explicitly set "DefaultRoute=false" we should do the right thing. That can either be return an error in the validation or set it correctly here.

Reply:
should be addressed!